script/are_kibana_saved_objects_updated.py

```diff
@@ -96,6 +97,25 @@ def iterate(val_id, key, v1, v2):
             print("---")
     return ret_val
 
+def get_kibana_commit(branch):
+    """
+    Looks up an open PR in Kibana against `branch`, and with 'apm', 'update', and 'index pattern' in the title (case insensitive).
+    If found, it is assumed to be a PR updating the Kibana index pattern - so this tests compares the content against
+    the one in that PR
+    Limitations:
+        - `index_pattern.json` must be found in HEAD (so in case of being amended, it needs to be force-pushed)
+        - returns the last PR open, which might be wrong if there are several updated at a time.
```

Comment:
Would it be possible to match by branch names to circumvent this limitation?

Reply:
oh, that comment is wrong actually. this matches the branch already (https://github.com/elastic/apm-server/pull/1860/files#diff-8f30b9f4f4c8eee6d6d642f585f9af2fR113), so the issue if there are several PRs against the same branch. ill update